src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,39 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int supplySum = 0;
+        int buySum = 0;
+        try {
+            List<String> lines = Files.readAllLines(Paths.get(fromFileName));
+            for (String line : lines) {
+                String[] parts = line.split(",");
+                if (parts[0].equals("supply")) {
```

Comment:
It's better to use constants for repeated strings such as 'supply' and 'buy' to improve code readability and maintainability.